README.md

```diff
@@ -89,6 +89,32 @@ iex> D.div(D.new(31), D.new(2))
 #Decimal<15>
 ```
 
+### Comparisons
+
+Using compare operators (`<`, `=`, `>`) directly in two decimals may not return
+the correct result. Instead use comparison functions.
+
+```elixir
+iex> D.compare(D.new(-1), D.new(0))
+#Decimal<-1>
```

Comment:
I think `D.compare` isn't super useful in practice, maybe let's not advertise it in the readme, or just briefly mention it? E.g.: "Also see `Decimal.compare/2`"

Reply:
removed it